doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -786,6 +785,7 @@ are applied using the tactic :n:`rewrite_strat @rewstrategy`.
    | eval @red_expr
    | fold @one_term
    | ( @rewstrategy )
+   | choice {+ @rewstrategy }
```

Comment:
Would you move this line to the same spot as the previous "choice"?  doc_grammar is not always able to preserve ordering info that's obvious to humans.

Reply:
Also, would you update this text 30-40 lines below to reflect the grammar change?
`left_biased_choice` (ugh) could be something like "use the first successful strategy".

```
:n:`choice @rewstrategy @rewstrategy`
   left_biased_choice
```